sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -66,6 +66,12 @@ struct asoc_sdw_codec_info codec_info_list[] = {
 				.dai_name = "tas2783-codec",
 				.dai_type = SOC_SDW_DAI_TYPE_AMP,
 				.dailink = {SOC_SDW_AMP_OUT_DAI_ID, SOC_SDW_AMP_IN_DAI_ID},
+				.init = asoc_sdw_ti_amp_init,
+				.rtd_init = asoc_sdw_ti_spk_rtd_init,
+				.controls = maxim_controls,
+				.num_controls = ARRAY_SIZE(maxim_controls),
+				.widgets = maxim_widgets,
+				.num_widgets = ARRAY_SIZE(maxim_widgets),
```

Comment:
I would move the commits before `ASoc: tas2783A: add machine driver changes`. So that we can add the above in the `ASoc: tas2783A: add machine driver changes` commit.